### PR TITLE
feat: [M3-8703] - Disable Create VPC button with tooltip text on empty state Landing Page for restricted users

### PR DESCRIPTION
**Pull request by hasyed-akamai**
## Description 📝
To prevent unauthorized access to specific flows and provide clearer guidance, we aim to restrict entry to users without the required permissions.

Here, we are restricting users from creating new VPC from the Empty State Landing Page when they do not have access or have read only access.

## Changes  🔄
List any change relevant to the reviewer.

- For restricted users:
  - Disabled Create VPC Button on the Empty Landing Page

## Preview 📷

| Before | After |
| ------ | ----- |
| Before | After |

### Prerequisites
- Log into two accounts side by side:
  - An unrestricted admin user account: full access
  - A restricted user account (use Incognito for this)
    - Start with Read Only for everything

### Reproduction steps
- Landing:
  - Observe as restricted user, notice shows and you cannot create VPC

### Verification steps

- After changes, observe tooltips are tailored to the action.

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/.changeset/pr-11052-added-1728305995058.md
Comment (hkhalil-akamai) [inline packages/manager/.changeset/pr-11052-added-1728305995058.md:1]: This change might fit better under "Changed"

**Review by hkhalil-akamai — APPROVED** [review]
Looks good! One minor suggestion regarding the changeset type.

File: packages/manager/.changeset/pr-11052-added-1728305995058.md
Comment (hasyed-akamai) [reply packages/manager/.changeset/pr-11052-added-1728305995058.md:1]: modified the changeset type to "Changed"

**Review by harsh-akamai — APPROVED** [review]
PR looks good! 🎉

**Pull request merged**